src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -1625,7 +1626,11 @@ namespace winrt::Microsoft::Terminal::Control::implementation
             _terminal->MultiClickSelection(terminalPosition, mode);
             selectionNeedsToBeCopied = true;
         }
-        _updateSelection();
+        _renderer->TriggerSelection();

```

Comment:
This calls into question the purpose of a function named `_updateSelection` if it is not supposed to be used in this case... right?